snapcraft/how-to/interface-management.md

```diff
@@ -1,26 +1,21 @@
 [Interfaces](/t/35928) allow (or deny) access to a resource outside of a snap's confinement.
 
-Most users don't need to worry about interfaces. Snaps are designed for strong application isolation and safe interface connections are made automatically. 
+Most users don't need to worry about interfaces. Snaps are designed for strong application isolation and safe interface connections are made automatically.
 
 An interface is most commonly used to enable a snap to access sound playback or recording, your network, and your $HOME directory. But which interfaces a snap requires, and *provides*, is very much dependent on the type of snap and its own requirements.
 
 See [Supported interfaces](/t/supported-interfaces/7744) for a comprehensive list of interfaces and what kind of access they permit.
 
-<h2 id='heading--slots-plugs'>Plugs and slots</h2>
+## View all the snaps using an interface
```

Comment:
This could be interpreted as _using an interface to view all the snaps _.  Maybe some more like _View the interfaces a snap is using_ ?